src/main/scala/beam/replanning/utilitybased/ChangeModeForTour.scala

```diff
@@ -79,7 +79,7 @@ class ChangeModeForTour(
               if (alternativesForTour.contains(CAR)) DRIVE_TRANSIT
               else WALK_TRANSIT,
               timeDist._1,
-              timeDist._2,
+              0.0,
```

Comment:
why?

Reply:
This parameter `time` is never used. So I decided to replace it with occupancyLevel.
Now I see that it's better to return back 'time' so that some ModeChoiceCalculators could use it.